core/src/main/scala/org/apache/spark/HeartbeatReceiver.scala

```diff
@@ -79,13 +79,19 @@ private[spark] class HeartbeatReceiver(sc: SparkContext, clock: Clock)
   private val executorTimeoutMs =
     sc.conf.getTimeAsMs("spark.storage.blockManagerSlaveTimeoutMs",
```

Comment:
This can use STORAGE_BLOCKMANAGER_SLAVE_TIMEOUT now; it does already in `master`. while technically you don't need to rebase this PR, it might be useful here so that we can see the current state of `master` vs your changes.